src/engine/commands/migration/commands/commit.ts

```diff
@@ -40,10 +43,28 @@ export default {
         ? await uploadProject(context, options)
         : { buildName: null };
 
+    if (params.mode === CommitMode.ONLY_PROJECT || params.mode === CommitMode.FULL) {
+      var migrationNames: string[] = [];
```

Comment:
```suggestion
   let migrationNames: string[] | undefined = undefined; 
    if (params.mode === CommitMode.ONLY_PROJECT || params.mode === CommitMode.FULL) {
      migrationNames = [];
```

Reply:
please don't use var

Reply:
Because of the global scope ? Or are there other reasons

Reply:
yes